docs/standard/microservices-architecture/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Use HttpClientFactory to implement resilient HTTP requests
-description: HttpClientFactory is an opinionated factory, available since .NET Core 2.1, for creating `HttpClient` instances to be used in your applications. 
+description: Resiliency | HttpClientFactory is an opinionated factory, available since .NET Core 2.1, for creating `HttpClient` instances, making it easy for you to include  to be used in your applications. 
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 07/03/2018
+ms.date: 16/10/2018
 ---
 # Use HttpClientFactory to implement resilient HTTP requests
 
```

Comment:
not related to you PR but the proper link to HttpClient in docs would be `<xref:System.Net.Http.HttpClient>`

Reply:
Added, still missing the criteria, since I asked for it about a couple of hours ago 😂.